assets/js/vue-apps/grant-filters.js

```diff
@@ -10,80 +11,145 @@ function init() {
         el: mountEl,
         delimiters: ['<%', '%>'],
         data() {
-            let facets = {};
+            // Populate data from global object (eg. to share server/client state)
+            let PGS = window._PAST_GRANTS_SEARCH;
+            let queryParams = PGS && PGS.queryParams ? PGS.queryParams : {};
+            let existingFacets = PGS && PGS.facets ? PGS.facets : {};
+            let existingSort = PGS && PGS.sort ? PGS.sort : {};
             let defaultFilters = {
                 amount: '',
                 year: '',
                 programme: '',
                 country: '',
                 localAuthority: ''
             };
-            let filters = Object.assign({}, defaultFilters);
             return {
-                facets,
                 defaultFilters,
-                filters
+                sort: Object.assign({}, existingSort),
+                facets: Object.assign({}, existingFacets),
```

Comment:
Do we need to assign when there is only one object?

Reply:
This was to clone them rather than extend anything, have been bitten in the past by mutation when just doing plain `facets = existingFacets`.